core/arch/arm/pta/device.c

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (C) 2019, Linaro Limited
+ */
+
+/*
+ * This pseudo TA is used by Linux TEE driver to fetch pseudo/early TA's UUIDs
```

Comment:
Drop "/early" (the below code does not enumerate early TAs).

Actually, I think it would be good to also enumerate early TAs, because a kernel service could very well be implemented in user-mode. Even TAs stored in the REE FS could be considered as soon as they are accessible... (tee-supplicant running). However, implementing this would be significantly more complicated (the TA flags are a bit more difficult to access as they are in a ELF segment, you have to use the TA store interface...). Could be a future extension if/when needed.